sys/shell/commands/sc_cryptoauthlib.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for the cryptoauthlib module
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include "cryptoauthlib.h"
+
+void get_bin(char *result, uint8_t byte)
+{
+    for (int i = 0; i < 8; i++) {
+        result[i] = (byte << i & 0x80 ? '1' : '0');
```

Comment:
cppcheck complains here. Not 100% sure but this might fix it: `((byte << i) & 0x80) ? '1' : '0'`